devito/builtins.py

```diff
@@ -53,6 +54,130 @@ def smooth(f, g, axis=None):
         dv.Operator(dv.Eq(f, g.avg(dims=axis)), name='smoother')()
 
 
+def gaussian_smooth(f, sigma=1, _order=4, mode='reflect'):
+    """
+    Gaussian smooth function.
+    """
+    class ObjectiveDomain(dv.SubDomain):
```

Comment:
Remind me again why this can't be (yet?) moved to `grid.py` along with DOMAIN and INTERIOR ?

Reply:
Hmm, it possibly can now that the 'reflection' bc is being dealt with by an `Operator`. Will check.